src/components/WalletSelect.vue

```diff
@@ -37,6 +37,11 @@ export default {
                       <img src="@/assets/wallets/lightning.png" class="object-contain w-full h-full"></img>
                    </button>
                 </li>
+                <li>
```

Comment:
I think we should think carefully about the optics of having VALR introduce us to a merchant and then having their logo be right at the bottom of the payment options.

Reply:
I have reached out to VALR to understand whether this would be a concern